importers/djornl/parser.py

```diff
@@ -709,13 +725,94 @@ def summarise_dataset(self):
                 'cluster': len(node_data['cluster']),
                 'full': len(node_data['full'])
             },
+            'errors_total': len(errs),
+            'errors': errs
         }
 
 
-if __name__ == '__main__':
+def format_summary(summary, output):
```

Comment:
you could move this into the class so that it's a parser method

Reply:
I'd also add the text/json switch to the `load_data` method, so that you can get the same capabilities running the parser in a script (e.g. `parser = DJORNL_Parser()` and then `parser.load_data(args)`) that you can from running it from the command line.